src/ocean/text/convert/Formatter_test.d

```diff
@@ -543,3 +543,10 @@ unittest
     const bool NO  = false;
     test(format("{} -- {}", YES, NO) == "true -- false");
 }
+
+// Test for a previous parser bug
```

Comment:
Was this regarded as a bug then?

Reply:
Not exactly. More like an unexpected and baseless limitation. I'll update the comment.

Reply:
Done